### DISCUSSION TITLE
No "Visits" for me

### DISCUSSION THREAD

**Discussion by shaman007** [Freika/dawarich#515] [Q&A] [answered]
**Describe the bug**

Despite I have the instance run for about a month, there is no "visits" suggested, etc. Just empty page.  How to trigger that job manually to get logs?

**Version**
0.19.3

**Comment by Freika** [comment, marked as answer]
Currently, the background job to suggest visits is disabled due to feature instability. I will return to it later and enable it back when it will work better. For now, the feature should be considered unusable and I'll probably move link to the page to a dropdown or something.

One thing that is useful is area visits. You can enable Areas layer on the map, draw and save an area and every midnight a background job will check if you have visited an area and suggest a visit there. The docs on the website will be updated with this information this week

**Comment by shaman007** [comment]
Thanks for clarification!
